tests/integration/conftest.py

```diff
@@ -119,61 +117,11 @@ def mitmdump_venv(shared_integration_test_tmpdir):
 
 
 @pytest.fixture
-def run_proxy(
```

Comment:
I factored this out to `testing/mitmproxy.py` to re-use mitmproxy in the new test in its reverse proxy mode to wrap PyPI / devpi with required auth.